ui-kit/src/main/kotlin/org/p2p/uikit/organisms/seedphrase/SecretKey.kt

```diff
@@ -0,0 +1,6 @@
+package org.p2p.uikit.organisms.seedphrase
+
+data class SecretKey(
+    val text: String = "",
+    val isValid: Boolean = true
```

Comment:
why it's valid by default?

Reply:
Because if not, it will be red by default.
We suppose that user enters the correct keys and then check for validity